### PR TITLE
Add new isEqualTo with auth for secured endpoints.

### PR DESCRIPTION
**Pull request by jsfrench**
Added a new isEqualTo method which supports authorization for accessing Swagger definitions stored at non-public URLs, e.g. private APIs on SwaggerHub.
It may be better not to require io.swagger.models.auth.AuthorizationValue as input to the assertj-swagger API, see what you think anyway.

### CODE REVIEW

File: src/main/java/io/github/robwin/swagger/test/SwaggerAssert.java [hunk 1]
```diff
@@ -85,6 +87,18 @@ public SwaggerAssert isEqualTo(String expectedLocation) {
...
 
+    /**
+     * Verifies that the actual value is equal to the given one.
+     * @param expectedLocation the location of the given value to compare the actual value to.
+     * @param auths List of io.swagger.models.auth.AuthorizationValue for access to the expectedLocation
+     * @return {@code this} assertion object.
+     * @throws AssertionError if the actual value is not equal to the given one or if the actual value is {@code null}..
+     */
+    public SwaggerAssert isEqualTo(String expectedLocation, List<AuthorizationValue> auths) {
+
```
Comment (marcelstoer) [inline src/main/java/io/github/robwin/swagger/test/SwaggerAssert.java]: Why the extra blank line?

File: src/main/java/io/github/robwin/swagger/test/SwaggerAssert.java [hunk 2]
```diff
@@ -85,6 +87,18 @@ public SwaggerAssert isEqualTo(String expectedLocation) {
...
     }
 
+    /**
+     * Verifies that the actual value is equal to the given one.
+     * @param expectedLocation the location of the given value to compare the actual value to.
+     * @param auths List of io.swagger.models.auth.AuthorizationValue for access to the expectedLocation
+     * @return {@code this} assertion object.
+     * @throws AssertionError if the actual value is not equal to the given one or if the actual value is {@code null}..
+     */
+    public SwaggerAssert isEqualTo(String expectedLocation, List<AuthorizationValue> auths) {
```
Comment (marcelstoer) [inline src/main/java/io/github/robwin/swagger/test/SwaggerAssert.java]: Why only for `isEqualTo ` but not for the others?

**Comment by marcelstoer** [comment]
Why only for `isEqualTo` but not for the others?

**Comment by john-french** [comment]
Thanks for comments @marcelstoer, update covers `satisfiesContract` also, which I think is all?

**Comment by jsfrench** [comment]
Thanks for comments @marcelstoer, update covers satisfiesContract also, which I think is all?

**Pull request merged**